data/json/effects.json

```diff
@@ -1761,6 +1779,7 @@
         "type": "effect_type",
         "id": "irradiated",
         "name": [ "Irradiated" ],
+        "desc": [ "You suffer symptoms of a radiation sickness that weakens your body." ],
```

Comment:
I don't know how to approach this one, since there really isn't a [way for how radiation would feel](https://www.quora.com/How-does-it-feel-when-we-are-exposed-to-nuclear-radiation), especially considering #24233.

Perhaps something a bit more abstracted... maybe `You feel random bouts of heat in your body and waves of nausea come and go.`

Reply:
That is a a problem, yes. Ideally this should be invisible to the player. There are ways to measure radiation, so maybe it should not be? A person can feel the symptoms, but would you even know that it's radiation sickness and not some other disease? Maybe in later stages if you knew exactly what happens in what order. Shall we consider removing it from view? Or maybe there should be a condition based on first aid skill to check if you have that knowledge?

Reply:
I'm of the mind that it should be hidden from player (just like teleglow sickness), and the only time it should pop up on `@` character screen is when a radiation device has been used recently (geiger counter, radiation biomonitor) that disappears after X time in again (3-5 minutes, so there's an actual reason to use these devices again and make the Integrated Dosimeter CBM more useful). The reason I say it should disappear because I can see players getting confused and annoyed ("But the irradiation status said it was less than 100, so I should've been safe!") Even a small line maybe at the end (`You'll need to check your radiation levels periodically to stay up to date` or something).

All this may be best saved for a separate PR, though I do appreciate your diligence, nexus!